sdk/communication/Azure.Communication.Calling.Server/src/CallClient.cs

```diff
@@ -0,0 +1,491 @@
+        public virtual Response<CancelAllMediaOperationsResponse> CancelAllMediaOperations(string callLegId, string operationContext = null, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(CancelAllMediaOperations)}");
+            scope.Start();
+            try
+            {
+                return RestClient.CancelAllMediaOperations(callLegId, operationContext, cancellationToken: cancellationToken);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary> Play audio in the call. </summary>
+        /// <param name="callLegId"> The call leg id. </param>
+        /// <param name="audioFileUri"> The uri of the audio file. </param>
+        /// <param name="loop">The flag to indicate if audio file need to be played in a loop or not.</param>
+        /// <param name="audioFileId">Tne id for the media in the AudioFileUri, using which we cache the media resource. </param>
+        /// <param name="callbackUri">The callback Uri to receive PlayAudio status notifications. </param>
+        /// <param name="operationContext">The operation context. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="audioFileUri"/> is null. </exception>
+        public virtual async Task<Response<PlayAudioResponse>> PlayAudioAsync(string callLegId, Uri audioFileUri, bool? loop, string audioFileId, Uri callbackUri, string operationContext = null, CancellationToken cancellationToken = default)
+            => await PlayAudioAsync(callLegId, new PlayAudioOptions { AudioFileUri = audioFileUri, Loop = loop, AudioFileId = audioFileId, CallbackUri = callbackUri, OperationContext = operationContext }, cancellationToken).ConfigureAwait(false);
+
+        /// <summary> Play audio in the call. </summary>
+        /// <param name="callLegId"> The call leg id. </param>
+        /// <param name="options"> Play audio request. </param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="options"/> is null. </exception>
+        public virtual async Task<Response<PlayAudioResponse>> PlayAudioAsync(string callLegId, PlayAudioOptions options, CancellationToken cancellationToken = default)
```

Comment:
Azure board suggest to using Option instead of Request. 
We use *Option as postfix for customer accessing model in apis and use *Request as postfix for model sent to Service in rest client